html-aam/roles.html

```diff
@@ -93,6 +93,7 @@ <h6 data-testname="el-h6" data-expectedrole="heading" class="ex">x</h6>
 
 <input type="button" value="x" data-testname="el-input-button" data-expectedrole="button" class="ex">
 <input type="checkbox" data-testname="el-input-checkbox" data-expectedrole="checkbox" class="ex">
+<input type="checkbox" switch data-testname="el-input-checkbox-switch" data-expectedrole="switch" class="ex">
```

Comment:
@scottaohara does that ID align with the hash link you intend to use in the HTML-AAM PR? If not, please suggest a replacement. The test names are intended to align with the spec IDs.

Reply:
FWIW, `el-input-checkbox-switch` looks good to me.

Reply:
This is what I would have used. So lgtm